rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction.adoc

```diff
@@ -0,0 +1,107 @@
+
+[[trackingsystem-resultentry-image]]
+image::as400ordertrackingscreen-searchresult-orderfound.png["Legacy order tracking system shows one search result.", 75%]
+
+Now you can copy the tracking number and reference it in your payment.
+
+Since all the scientists at Mythical University travel a lot, you have to repeat this tedious, error-prone and risky process several times per day. 
+
+You decide to get a bot to do this.   
+
+The following example shows how you automate the retrieving of the order ID from a PDF and of the tracking number from the legacy system using MuleSoft RPA as an invocable automation step. How to download the right attachment before and how to proceed afterwards isn't part of this example.
+
+## Design a Model of the Process in RPA Manager
+
+In RPA Manager, you open a new project and design a model of the process.
+
+The model contains the following activities:
+
+. Extract the order ID from the order confirmation.
+. Extract the corresponding tracking number from the legacy system.
+. Return either the tracking number or an information that it couldn't be found.
+
+image::extractrackingnumber-bpmn.png["In the flow, activities one and two are sequential. Depending on whether the order id is found, an exclusive gateway branches the flow to the third activity."]
+
+The user tasks in the third activity are for illustrational purposes and can be replaced by steps to prepare the returned data for processing in another automation step.
+
+After you finish the first draft of the model, you move the project to the build phase.
+
+Then, you open the project in RPA Builder to implement the workflows for the bot activities.
+
+## Create Activity Parameters in RPA Builder
+
+The process uses variables to store and route data. You define these _Activity Parameters_ in RPA Builder.
+
+For your process you need three Activity Parameters:
+
+* order_id
+** an alphanumeric variable
+** stores the order ID found in the order confirmation
+** is returned at the end of the process
+* tracking_no
+** an alphanumeric variable
+** stores the corresponding tracking number found in the order tracker application
+** is returned at the end of the process if it is found
+* is_order_id_found
+** a boolean variable
+** is set to true if the order id is found in the order tracking system
+** routes the process flow through the gateway accordingly
+
+image::activityparameters.png["The Activity Parameter tab in the Business Process Initialization window on the modelling workbench in RPA Builder shows the three parameters of the process."]
+
+To use Activity Parameters in a Workflow, move them from *Available Parameters* to *Used in this Workflow* in the *Activity Parameters* Action Step of the *Workflow Initialization* section. The parameters retain the values set in previous workflows throughout the process.
```

Comment:
Should Workflow be lowercase?

Reply:
No, as Activity Parameter, Workflow is a proper noun in MuleSoft RPA.

Reply:
Speaking of which, CrisV doesn't init cap Action Step. It would be good to get consistent on that. 

Our style is to favor lowercase (generic) unless it's a reference to something in the UI.